practice-two/src/javascript/main.js

```diff
@@ -1,2 +1,2 @@
-import { getNotes } from './utils/httpUtils'; 
-import { renderAllNotes } from './modules/getNotes';
+import {renderNote, renderAllNotes } from './modules/getNotes';
+import {renderNotesByCategory} from './modules/filterNotes'
```

Comment:
@quanghuy1799 You only do import and don't invoke the functions. I think it doesn't work as expected. Please be careful and make sure to test before requesting the review.